src/main/java/com/forgeessentials/chat/command/CommandTimedMessages.java

```diff
@@ -130,7 +124,7 @@ public String getPermissionNode()
     @Override
     public String getCommandUsage(ICommandSender sender)
     {
-        return "/automessage [select|broadcast|add|del] Select, broadcast, add or remove messages";
+        return "/TimedMessageHandler [select|broadcast|add|del] Select, broadcast, add or remove messages";
```

Comment:
isn't this supposed to be `/timedmessage ...` as in [Line 23](https://github.com/ForgeEssentials/ForgeEssentialsMain/pull/1532/files#diff-7fd9849db0f1ec7af179f816c44004a9R23)